source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -585,7 +585,7 @@ def _build_fallback(
     car_park_crowfly_duration = None
     via_pt_access = None
     via_poi_access = None
-    language = request.get('_asgard_language', "english_us")
+    language = request.get('language', "en-US")
```

Comment:
Same for https://github.com/hove-io/navitia/pull/4084/files#diff-1db49300dbe77e5d5ef58deb0ebd54793fd7851c5f9a3b238ebd4091567bf5caR133 and https://github.com/hove-io/navitia/pull/4084/files#diff-3307dc8a614b49809e0d0843ff12386f70537d2f093790b1d2872d6fd89ab70eR352:
Here you don't "respect" the default value of the coverage, no?
I think you should default to None and have asgard-connector deal with the default(coverage) value in a single place.

Reply:
It's nearly impossible to have language = None, and it arrives then the fallback language will be **en-US**